rosetta/docs/PGLE.md

```diff
@@ -8,9 +8,8 @@ The workflow to use the Profile Guided Latency Estimator with Nsight Systems (Ns
 1. **Profile Run**: We need to run the workload once, with the async collectives and latency hiding scheduler disabled. To be specific we need to turn the following three XLA flags off. All the remaining flags should be unchanged.
 
 ```
-export XLA_FLAGS="--xla_gpu_enable_latency_hiding_scheduler=false
---xla_gpu_enable_async_all_gather=false
---xla_gpu_enable_async_reduce_scatter=false"
+export XLA_FLAGS="--xla_gpu_enable_latency_hiding_scheduler=false"
```

Comment:
I think here we need `--xla_gpu_disable_async_collectives=allreduce,allgather,reducescatter,collectivebroadcast,alltoall,collectivepermute` to disable them all.

Reply:
I assume I should add these on-top instead of replacing the existing args, right?

Reply:
I believe that `--xla_gpu_disable_async_collectives=allgather,reducescatter` is the new way of spelling `--xla_gpu_enable_async_all_gather=false --xla_gpu_enable_async_reduce_scatter=false`.